zsh-aws-vault.plugin.zsh

```diff
@@ -16,44 +16,61 @@ alias avs='aws-vault server'
 # Convenience Functions                                              #
 #--------------------------------------------------------------------#
 function avl() {
-  aws-vault login $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login $aws_profile
 }
 
 function avll() {
-  aws-vault login -s $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login -s $aws_profile
+}
+
+function aven() {
+  local aws_profile=${1:-$AWS_VAULT}
+  unset AWS_ACCESS_KEY_ID
+  unset AWS_DEFAULT_REGION
+  unset AWS_REGION
+  unset AWS_SECRET_ACCESS_KEY
+  unset AWS_SECURITY_TOKEN
+  unset AWS_SESSION_EXPIRATION
+  unset AWS_SESSION_TOKEN
+  unset AWS_VAULT
+  export $(aws-vault exec $aws_profile -- env | grep AWS)
```

Comment:
I think we'll need to retain the case statement (`case ${AWS_VAULT_PL_MFA} in`) to support yubikey devices (see `avsh` and `avli` below).

Since that code's duplicated a few places, you might even consider breaking out a helper method.